cypress/support/pages/yves/multi-factor-auth/repositories/b2b-mp-multi-factor-auth-repository.ts

```diff
@@ -0,0 +1,51 @@
+import { injectable } from 'inversify';
+import { MultiFactorAuthRepository } from '../multi-factor-auth-repository';
+
+@injectable()
+export class B2bMpMultiFactorAuthRepository implements MultiFactorAuthRepository {
+  private readonly SELECTORS = {
```

Comment:
Why do we describe selectors like this?
please check other examples from other tests/classes that we already have and refactor these